tools/mipgen/src/main.cpp

```diff
@@ -193,14 +203,18 @@ int main(int argc, char* argv[]) {
     }
     Path inputPath(argv[optionIndex++]);
     string outputPattern(argv[optionIndex]);
-    if (!g_formatSpecified) {
+    if (Path(outputPattern).getExtension() == "ktx") {
+        g_ktxContainer = true;
+        g_formatSpecified = true;
+    } else  if (!g_formatSpecified) {
```

Comment:
nit: extra space between `else` and `if`

Reply:
Done